mix.exs

```diff
@@ -36,6 +43,9 @@ defmodule GuardianOnetime.Mixfile do
     ]
```

Comment:
Should we be marking postgrex as optional?

Reply:
maybe...we should probably test this against postgres and mysql 5.7 and verify. Then we can make both optional.